worker/instancepoller/worker.go

```diff
@@ -49,10 +101,47 @@ func (config Config) Validate() error {
 	return nil
 }
 
+type pollGroupType uint8
+
+const (
+	shortPollGroup pollGroupType = iota
```

Comment:
This one is only used internally to index the `pollGroup` field and is not printed out anywhere so I preferred to use a uint8 to represent it. I don't mind changing this to plain strings if you think it will aid code readability.